groovy-modifications/README.md

```diff
@@ -0,0 +1,89 @@
+
+The network element is added to the network and the switches are also created to connect the element 
+to a given busbar section with a closed disconnector and a breaker. Open switches are also added to every parallel busbar sections.
+
+Other similar functions are available to add every kind of injections and branches.
+
+# CreateLoadBay.groovy
+
+In this script, a node/breaker network is first loaded. Then a load is added on the busbar section BBS1 of the voltage level named S1VL2 and the network is exported.
+
+Before adding the load, the voltage level S1VL2 is:
+
+![S1VL2](.github/nb_network_s1vl2.png "S1VL2")
+
+The script is adding the load between the transformer TWT (order position 10) and the VSC converter station VSC1 (order position 20). For that, the order position is put to 15 when creating the modification.
+It is also possible to specify the direction of the load. Here it is not, so by default it is directed to the bottom.
+
+The voltage level S1VL2 will look like that after applying the modification:
+
+![S2VL1 with new load](.github/nb_network_with_load.png "S2VL1 with new load")
+
+It is possible to create a builder with any kind of injection adder and thus create any type of injection.
+
+Here are the options that must/can be filled:
+
+| Parameter              | Default value       | Description                                                                                                                                                                       |
+|------------------------|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| InjectionAdder         | /                   | The adder corresponding to the injection to be added. <br/>Should be created beforehand and contain all the required information.                                                 |
+| BusOrBusbarSectionId   | /                   | The ID of the busbar (node/breaker network) or of the bus (bus/breaker network) on which the injection should be connected.                                                       |
+| InjectionPositionOrder | /                   | Required in Node/Breaker. The order position of the injection to create the corresponding extension. <br/> The injection will be positioned accordingly on the busbar section.    |
+| InjectionFeederName    | Id of the injection | Only in Node/Breaker. An optional name to be put in the ConnectablePosition extension and that will be displayed on the diagrams.                                                 |
+| InjectionDirection     | BOTTOM              | Only in Node/Breaker. An optional direction to be put in the ConnectablePosition extension and that will correspond to the direction of the injection in the Single line diagram. |
+
```

Comment:
Add here a NB for people to say that the current design is not exactly groovy-friendly...